src/components/Playbar/index.tsx

```diff
@@ -23,6 +23,10 @@ import { ChangeHandler } from '@aiera/client-sdk/types';
 import './styles.css';
 
 function toDurationString(totalSeconds: number) {
+    if (totalSeconds == Infinity) {
```

Comment:
Don't think you need the explicit return here. Can just update the existing `if` statement below:

```
if (!totalSeconds || isNaN(totalSeconds) || Math.abs(totalSeconds) === Infinity) {
    totalSeconds = 0;
}
```

Not sure if it's needed, but I added `Math.abs` to account for `-Infinity`. Also, one thing to note, in JS, always use `===` vs `==`